plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreDataSource.java

```diff
@@ -497,7 +499,15 @@ protected Connection openConnection(@NotNull DBRProgressMonitor monitor, @Nullab
         if (instance != null) {
             log.debug("Initiate connection to " + getServerType().getServerTypeName() + " database [" + instance.getName() + "@" + conConfig.getHostName() + "] for " + purpose);
         }
+        boolean timezoneOverridden = false;
         try {
+            // Old versions of postgres and some linux distributions, on which docker images are made, may not contain
+            // new timezone, which will lead to the error while connecting, there is no way to know before connecting
+            // so to be sure we will use the old name
+            if ("Europe/Kyiv".equals(TimeZone.getDefault().getID())) { //$NON-NLS-1$
```

Comment:
Please move it to constant

Reply:
What if new PG server version will have a fix for this timezone?

Reply:
> What if new PG server version will have a fix for this timezone?

Even if the new version will contain a new timezone, it still requires Linux distributions on which it is installed to have this timezone as well. We can't just be sure based on postgres version.